forte/processors/data_augment/base_data_augment_processor.py

```diff
@@ -58,20 +58,17 @@ def modify_index(
     r"""
     A helper function to map an index before replacement
     to the index after replacement.
-
```

Comment:
It might be better to keep the empty lines there as they separate docstring paragraphs clearly.